### PR TITLE
disabled property editing button and display empty fields

### PR DESCRIPTION
**Pull request by lordstone**
This PR does the following tasks:

1. display all fields now, for the missing items, it will show empty or default value in the fields.
2. disabled the editing properties feature globally, so user will not be able to directly add/drop fields.

TODO: fix the raw JSON remove issue

### CODE REVIEW

File: src/editors/object.js [hunk 1]
```diff
@@ -816,22 +819,42 @@ JSONEditor.defaults.editors.object = JSONEditor.AbstractEditor.extend({
...
+          self.addObjectProperty(i);
+          if(self.editors[i]) self.editors[i].setValue(value[i],initial);
+        } else {
+          window.console.log('try adding:' + i);
+        }
+      }
+    });
+*/
+
+    // something that add the missing schema properties
```
Comment (Syzygy24) [inline src/editors/object.js]: Should we remove this comment? Is this part ready?

**Review by Syzygy24 — COMMENTED** [review]
If I have a schema with an "age" field with "default": 25
and I take "age" out of the data, the form displays nothing, but the json output is a 0. Would we want to display / return the schema's default value in that case?

File: src/editors/object.js (hunk 1 above)
Comment (lordstone) [reply src/editors/object.js]: Yes, if we don't want to go back to the state where it only shows up fields that are present in data, we can remove those comments

**Comment by Syzygy24** [comment]
There are some merge conflicts after merging the last pr. 
We also might want to update the minor version number

**Comment by lordstone** [comment]
Merged here: https://github.com/Syzygy24/y-jsoneditor/pull/9

**Pull request closed without merging**